mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/CompletionRecord.java

```diff
@@ -0,0 +1,69 @@
+package org.motechproject.nms.mobileacademy.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Tracks the completion record for a given calling number
+ */
+@Entity(tableName = "nms_ma_completion_records")
+public class CompletionRecord {
+
+    @Field
+    private long callingNumber;
```

Comment:
#118 added validation around our storing of calling numbers that you'll need to add here.

    @Min(value = 1000000000L, message = "callingNumber must be 10 digits")
    @Max(value = 9999999999L, message = "callingNumber must be 10 digits")
    @Column(length = 10)

Ideally there would also be some unit tests to validate it.